main.go

```diff
@@ -45,39 +45,43 @@ func main() {
 		os.Getenv("DEBUG") == "true",
 	)
 
-	mc := &mail.Client{Config: mailCfg.NewFromEnv()}
+	// Initialize database
 	db, err := database.New(dbCfg.NewFromEnv())
 	if err != nil {
 		slog.Error("failed to initialize database", slog.Any("error", err))
 		panic(err)
 	}
+	if err := db.Migrate(&models.User{}); err != nil {
+		slog.Error("failed to migrate database", slog.Any("error", err))
+		panic(err)
+	}
+
 	apiClient := server.Client{
 		Config:      serverCfg.NewFromEnv(),
 		RateFetcher: rate.NewNBURateFetcher(),
-		DB:          db,
-	}
-	ctx = context.WithValue(ctx, settings.APIClientKey, apiClient)
-	apiClient.Engine = server.NewEngine(ctx)
-
-	if err := apiClient.DB.Migrate(&models.User{}); err != nil {
-		slog.Error("failed to migrate database", slog.Any("error", err))
-		panic(err)
+		UserRepo:    *models.NewUserRepository(db),
 	}
 
+	// Start cron job for notifications
 	cronSpec := os.Getenv("CRON_SPEC")
```

Comment:
Can wt localise all work with env variable in one place? Near the .env.example file for example.